src/main/java/seedu/hirehub/logic/commands/AddApplicationCommand.java

```diff
@@ -39,7 +39,7 @@ public class AddApplicationCommand extends Command {
     public static final String MESSAGE_EXCEEDS_VACANCY = "The number of accepted candidates already meets the"
             + " stipulated vacancy.\n To accept more candidates, the vacancy for the job can be increased via the"
             + " edit_job command, or change status for existing application(s) to this job via status command to"
-            + " a status other than ACCEPTED. You can retrieve vacancy left via slots_left command";
+            + " a status other than ACCEPTED.\n You can retrieve vacancy left via slots_left command";
```

Comment:
Should the whitespace after "\n" be there? Should it be "vacancies" instead of "vacancy"?